File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-api/src/main/java/com/cbi/ccr/csw/dashboard/dto/ServiceRegistryDTO.java
package com.cbi.ccr.csw.dashboard.dto;

import java.time.LocalDateTime;
import java.util.Set;

import com.cbi.ccr.csw.domain.csw.ServiceStatus;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class ServiceRegistryDTO {
	private String id;
	private String groupId;
	private Set<String> roles;
	private LocalDateTime lastUpdate;
	private ServiceStatus serviceStatus;
	private String hostName;
	private Integer port;
}

File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-ui/src/app/modules/configuration/service-registry/dto/service-registry-dto.ts
import { CommonConfigDTO, CommonConfigFilterDTO } from "../../common-config-dto";

export class ServiceRegistryFilterDTO extends CommonConfigFilterDTO{

    propertyName!: string;
  
}

export class ServiceRegistryDTO extends CommonConfigDTO{
    groupId!: string;
    hostName!: string;
    lastUpdate!: string;
    port!: number;
    roles!: string[];
    serviceStatus!: string;
}